api4/cloud.go

```diff
@@ -486,6 +487,22 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = appErr
 			return
 		}
+	case model.EventTypeSubscriptionChanged:
+		// event.ProductLimits is nil if there was no change
+		if event.ProductLimits != nil {
+			c.App.GetPluginsEnvironment()
+			if pluginsEnvironment := c.App.GetPluginsEnvironment(); pluginsEnvironment != nil {
+				pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
+					hooks.OnCloudLimitsUpdated(event.ProductLimits)
+					return true
+				}, plugin.OnCloudLimitsUpdatedID)
+			}
+		}
+		if err := c.App.Cloud().UpdateSubscriptionFromHook(event.ProductLimits, event.Subscription); err != nil {
+			c.Err = model.NewAppError("Api4.handleCWSWebhook", "api.cloud.subscription.update_error", nil, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		c.Logger.Info("Updated subscription")
```

Comment:
Was this intended to stay , maybe:
```suggestion
		c.Logger.Info("Updated subscription from webhook event")
```